integration/securitypolicy_test.go

```diff
@@ -197,16 +293,15 @@ func TestSecurityPolicyMigration(t *testing.T) {
 					copySpec := policyCR.Spec.DeepCopy()
 					copySpec.Name = "Updating Existing Policy"
 
-					err = klient.Universal.Portal().Policy().Update(reqCtx, copySpec)
+					err = updatePolicyOnTyk(reqCtx, copySpec)
 					eval.NoErr(err)
 
 					// Ensure that policy is updated accordingly on Tyk Side.
 					newCopySpec, err := klient.Universal.Portal().Policy().Get(reqCtx, policyCR.Status.PolID)
 					eval.NoErr(err)
```

Comment:
Probably you want to retry if policy is not created yet? In that case, you need to return false. Because eval will fail the test instead of retrying.

Reply:
yes you are right. I updated as you suggested 👍